routing.rst

```diff
@@ -37,15 +37,22 @@ Otherwise, create the following file manually:
     # config/routes/attributes.yaml
     controllers:
         resource: ../../src/Controller/
-        type: attribute
+        type: attribute@App\Controller
 
     kernel:
-        resource: ../../src/Kernel.php
+        resource: App\Kernel
         type: attribute
 
-This configuration tells Symfony to look for routes defined as
-attributes in any PHP class stored in the ``src/Controller/``
-directory.
+This configuration tells Symfony to look for routes defined as attributes on
+classes declared in the ``App\Controller`` namespace which are stored in the
+``src/Controller/`` directory which follows the PSR-4 standard. In addition,
```

Comment:
missing colon `directory, which follows`?

Reply:
Not sure, it feels very German to set that comma. Let's see what the docs team says.